packages/interface-adapter/lib/adapter/types.ts

```diff
@@ -18,6 +19,18 @@ export type BlockType = EvmBlockType | any;
 export type Transaction = EvmTransaction | any;
 export type TransactionReceipt = EvmTransactionReceipt | any;
 export type TxHash = string;
+export type TransactionReportData = {
```

Comment:
What if we got rid of `Data`? How's just `TransactionReport`? or maybe `TransactionCostReport` if we wanted to be more explicit that way.

Reply:
Renamed to `TransactionCostReport`.